### DISCUSSION TITLE
PatchTool 6.4 6/8

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#352] [Announcements]
更新至中國服6.31，修正部分錯誤。

部分技能、狀態、對話因與 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/259 中提到的相同原因，目前會顯示原文。

更新漢化程式，移除未使用的library，減少程式大小。

## 直接用

新增漢化檔案，解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20230608.rar

## 其他檔案

檔案：

- FFXIVChnTextPatch-20230608.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230608.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- FFXIVChnTextPatch-20230603.jar：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20230608.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

## 附註
屬於6.31內容，但仍會顯示原文的技能、對話、狀態：
* Action.csv: 31544, 31545, 31550, 31551, 31624, 31648-31656, 31660, 31664-31670, 32788, 32789
* BNpcName.csv: 12256, 12259, 12260
* InstanceContentTextData.csv: 35810-35821, 35823, 35825, 35827-35837
* Status.csv: 3427-3429, 3433, 3435, 3439, 3442-3444, 3446-3448, 3452, 3453, 3532

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.1.1.0'>PatchTool 6.4 6/8</a>.</em>

**Comment by Bos-hunting** [comment]
請問今天國際服6.41的更新可以直接沿用當前的這個漢化包嗎？

**Reply by GpointChen** [reply]
有部分文本有更改，但改得不多；使用v3.1.1.0漢化器還原後重新漢化理論上不會有大問題。

不過我還是更新了v3.1.1.1，建議使用最新版。
